integration/jnifuse/fs/src/main/java/alluxio/jnifuse/FuseFileSystem.java

```diff
@@ -15,6 +15,9 @@
 import alluxio.jnifuse.struct.FuseContext;
 import alluxio.jnifuse.struct.FuseFileInfo;
 import alluxio.jnifuse.struct.Statvfs;
+import jnr.ffi.Pointer;
```

Comment:
wrong import session?
https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#coding-style

Reply:
remove it